src/components/ForgotPassword.vue

```diff
@@ -43,7 +43,7 @@
 
               <div class="flex items-center justify-between">
                 <div class="flex items-center">
-                  <Button type="submit" :on-click="consoleClick">
+                  <Button type="submit"  @emitclick="consoleClick">
```

Comment:
`emitclick` should be `click`

Reply:
@giteshk11 I got the issues. I will fix it and let you know.